packages/wonder-blocks-dropdown/src/components/__tests__/dropdown-core-virtualized.test.js

```diff
@@ -24,6 +26,12 @@ describe("DropdownCoreVirtualized", () => {
     });
 
     afterEach(() => {
+        // We have to explicitly unmount before clearing mocks, otherwise jest
+        // timers will throw because we'll try to clear an animation frame that
+        // we set with a setTimeout but are clearing with clearAnimationFrame
+        // because we restored the clearAnimationFrame mock (and we won't
+        // have cleared the timeout we actually set!)
+        unmountAll();
```

Comment:
Because of what the comment says.

```
        // We have to explicitly unmount before clearing mocks, otherwise jest
        // timers will throw because we'll try to clear an animation frame that
        // we set with a setTimeout but are clearing with clearAnimationFrame
        // because we restored the clearAnimationFrame mock (and we won't
        // have cleared the timeout we actually set!)
```